src/components/home/employees/EmployeeFilter.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import debounce from 'lodash/debounce';
+
+const FILTER_SEARCH_DELAY = 300;
+const keys = {
+  ENTER: 13,
+};
+
+class EmployeeFilter extends Component {
+  onKeyDownHandler = e => {
```

Comment:
let's just call this `onKeyDown`
```suggestion
  onKeyDown = e => {
```